lib/src/rules/unnecessary_null_aware_operator_on_extension_on_nullable.dart

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:analyzer/dart/element/element.dart';
+
+import '../analyzer.dart';
+
+const _desc =
+    r'Unnecessary null aware operator on extension on a nullable type';
+
+const _details = r'''
+
+Avoid null aware operator for member defined in an extension on a nullable type.
```

Comment:
? => 

Avoid null aware operators for members defined in an extension on a nullable type.